canonical-web-and-design/ubuntu.com
# Update /internet-of-things/appstore

## Done

Updated /internet-of-things/appstore according to [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/5994#issuecomment-546304958) and [copy doc](https://docs.google.com/document/d/1Wj-aq6qPnNMjgMY3yyBeJRjbBVVCRBq6s00JkR6-d_A/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/appstore
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/5994#issuecomment-546304958) and [copy doc](https://docs.google.com/document/d/1Wj-aq6qPnNMjgMY3yyBeJRjbBVVCRBq6s00JkR6-d_A/edit#)

## Issue / Card

Fixes #5994
